infra/README.md

```diff
@@ -1,8 +1,16 @@
-## Fetching Debian Bullseye Image and Verifying It
 
-- `./base-image/fetch-debian-image.sh`
+## Environment choice
+This section covers how to set a production-like environment. 
+For most developments, it is better to use the containers instead (see the dev-env directory). The infrastructure environment provides additional services like reverse-proxy (nginx) and monitoring, but is also usually not necessary and less practical for backend and frontend development tasks.
 
-## Installing in a VM for Infrastructure Code Development
+There are 2 options, renting a server to a cloud provider, or creating a test VM for development:
+- Setting up a VM (development only). It is potentially safer, since it is free and not exposed to the internet. This may significantly slow your computer on runtime. A specific network configuration like port forwarding is also needed to communicate with the VM.
+- Renting a server in the cloud is closer to the production environment. Most cloud providers (Vultr, Digital Ocean, AWS...) offer free credit for new users, but it can get costly if you rent a powerful server and forget to stop it after use.
+
```

Comment:
I suggest to remove this whole part also. The extra information contained in the " Renting " a server paragraph can be move below. See my next suggestions.

```suggestion
```